index.js

```diff
@@ -4,6 +4,7 @@ import getHomeDirectory from '#home-directory';
 
 const extractPathRegex = /\s+at.*[(\s](.*)\)?/;
 const pathRegex = /^(?:(?:(?:node|node:[\w/]+|(?:(?:node:)?internal\/[\w/]*|.*node_modules\/(?:babel-polyfill|pirates)\/.*)?\w+)(?:\.js)?:\d+:\d+)|native)/;
+const simplePathRegex = /^\w+\.js:\d+:\d+$/;
```

Comment:
This matches the end of `pathRegex`. So it does not need to cover other characters than `\w`, since `pathRegex` would not match then.